scapy/arch/windows/__init__.py

```diff
@@ -72,11 +72,39 @@ def _encapsulate_admin(cmd):
     # rights, which will execute the command
     return "Start-Process PowerShell -windowstyle hidden -Wait -Verb RunAs -ArgumentList '-command &{%s}'" % cmd
 
+def _hack_windows_subprocess():
+    """HACK: python 2.7 file descriptors.
+
+    This magic hack fixes https://bugs.python.org/issue19575
+    by adding HANDLE_FLAG_INHERIT to all already opened file descriptors.
```

Comment:
Please update the docstring to better reflect the implementation.
Something like:
"by suppressing passing HANDLE_FLAG_INHERIT to a range of already opened file descriptors."

Reply:
Nice catch, thanks